server/.envrc.example

```diff
@@ -14,7 +14,7 @@ export NODE_ENV=development
 export ENCRYPTION_PASSWORD_00001=somesecret
 
 # Options: prod, preprod, dev
-export MATH_ENV=dev
+export MATH_ENV=prod
```

Comment:
Wondering if we shouldn't consider a bit better config settings for dev/prod. Some other config related issues under consideration at the moment: https://github.com/compdemocracy/polis/pull/1587